dialect.go

```diff
@@ -310,7 +310,9 @@ func (m ClickHouseDialect) dbVersionQuery(db *sql.DB) (*sql.Rows, error) {
 }
 
 func (m ClickHouseDialect) insertVersionSQL() string {
-	return fmt.Sprintf("INSERT INTO %s (version_id, is_applied) VALUES ($1, $2)", TableName())
+	// Note, throughout the codebase is_applied is always true on inserts, so maybe hard-coding this
+	// is fine?
+	return fmt.Sprintf("INSERT INTO %s (version_id, is_applied) VALUES ($1, 1)", TableName())
```

Comment:
I hope this can be fixed upstream, so we can continue to use `bool` as we did before.

https://github.com/ClickHouse/clickhouse-go/issues/621

Reply:
if we ever want to do a breaking change (v5), I'd suggest removing is_applied column

Reply:
Agreed. I don't know what that would even look like, i.e., we might change the application code but how do we migrate the existing table.

Maybe we could tackle it as part of https://github.com/pressly/goose/issues/288